### PR TITLE
Allow running without spago

### PR DESCRIPTION
**Pull request by EMattfolk**
So... we sort of inlined the whole std lib. We don't need spago anymore

### CODE REVIEW

**Comment by EMattfolk** [comment]
I think we can remove quite a bit more code from the repo as well. I know `purserl` is mostly unused

**Comment by EMattfolk** [comment]
We should probably bump the major version number

File: lib/purserl.ex [hunk 1]
```diff
@@ -35,7 +35,7 @@ defmodule DevHelpers.Purserl do
     state = %{
       port: nil,
       caller: [],
-      purs_cmd: nil,
+      purs_cmd: "purs compile --codegen erl \"lib/**/*.purs\"",
```
Comment (FredTheDino) [inline lib/purserl.ex:38]: This won't work for people who have dependencies with Spago.

This fix will need to respect opting in and out of using Spago to be useful.

**Review by FredTheDino — CHANGES_REQUESTED** [review]

**Comment by FredTheDino** [comment]
Reworked this into https://github.com/drathier/purerlex/commit/2f1331d3e73084698d9434aa302e44d74d51429f

**Pull request closed without merging**